services/portfolios_ca/src/services/targets.ts

```diff
@@ -140,7 +144,33 @@ export default () => {
         name: 'Aguas - Rios',
       },
     ],
-  };
 
+    /**
+     * Get a raster layer by portfolio in a given area
+     *
+     * @param {String} areaType area type
+     * @param {String | Number} areaId area id
+     * @param {Number} portfolioId portfolio id
+     *
+     * @returns {Binary} result image with the portfolio cropped by search area
+     */
+    getPortfoliosCALayer: async (portfolioId: number) => {
```

Comment:
Do you know why all the async calls in this file have this suggestion:  `JSDoc types may be moved to TypeScript types.ts(80004)` ?

Reply:
no